Framework/Kernel/inc/MantidKernel/PropertyWithValue.h

```diff
@@ -12,6 +9,7 @@
 
 #ifndef Q_MOC_RUN
 #include <boost/lexical_cast.hpp>
+#include <boost/make_shared.hpp>
 #include <boost/shared_ptr.hpp>
```

Comment:
In my system, boost/make_shared.hpp seems to include boost/shared_ptr.hpp as well. Perhaps `#include <boost/shared_ptr.hpp>` here is superfluous, then?